code/script.js

```diff
@@ -6,14 +6,138 @@ alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
 
+const userName = prompt("Please enter your name:");
+alert(`Hello, ${userName}! Let's get started.`);
+
+
 // Step 2 - Food choice
 // Your code goes here
 
+const foodChoice = prompt(`Select your yummy food:
+
+1 - Pizza
+2 - Pasta
+3 - Salad
+
+Please enter the number of your choice:`);
+
+let selectedFood = "";
+
+if (foodChoice === "1") {
+  selectedFood = "Pizza";
+} else if (foodChoice === "2") {
+  selectedFood = "Pasta";
+} else if (foodChoice === "3") {
+  selectedFood = "Salad";
+} else {
+
+  alert("Invalid choice. Please select a valid number.");
+  process.exit(1);
+}
+
+alert(`You've chosen ${selectedFood}!`);
+
+
 // Step 3 - Subtype choice
 // Your code goes here
 
+let subFood = ""
+let foodNames = ""
+
+switch (selectedFood) {
+  case "Pizza":
+    subFood = prompt(`Select your Pizza type:
+
+1 - Margherita
+2 - Napolitana
+3 - Bianca 
+
+Please enter the number of your choice:`);
+    foodNames = ["Margherita", "Napolitana", "Bianca"];
+    break;
+
+  case "Pasta":
+    subFood = prompt(`Select your Pasta type:
+
+1 - Cacio e Pepe
+2 - Vongole
+3 - Carbonara 
+
+Please enter the number of your choice:`);
+    foodNames = ["Cacio e Pepe", "Vongole", "Carbonara"];
+    break;
+
+  case "Salad":
+    subFood = prompt(`Select your Salad type:
+
+1 - Pomodoro e basilico
+2 - Buratta
+3 - Melon e Peperoncino
+
+Please enter the number of your choice:`);
+    foodNames = ["Pomodoro e basilico", "Buratta", "Melon e Peperoncino"];
+    break;
+
+  default:
+    alert("Invalid choice.");
+    process.exit(1);
+}
+
+if (subFood === "1") {
+  alert(`Great choice, You've selected ${foodNames[parseInt(subFood) - 1]} for your ${selectedFood}!`)
+} else if (subFood === "2") {
+  alert(`Great choice, You've selected ${foodNames[parseInt(subFood) - 1]} for your ${selectedFood}!`)
+} else if (subFood === "3") {
+  alert(`Great choice, You've selected ${foodNames[parseInt(subFood) - 1]} for your ${selectedFood}!`)
+}
+
```

Comment:
I would like to see an "else" here that executes "Invalid choice". If I type e.g number 4 it just goes on to the next section asking for my age. Otherwise really good! :)